test/unit/typing.test.js

```diff
@@ -1046,4 +1046,39 @@ suite('typing with auto-replaces', function() {
       assert.equal(mq.typedText('n').latex(), 'x^{2n}');
     });
   });
+
+  suite('Chemistry Symbols', function() {
+    test('add bond via mq.write', function() {
+      mq.write('\\ce{\\bond{#}}');
+      assertLatex('\\ce{\\bond{#}}');
+    });
+    test('invalid bond types not added', function() {
+      mq.write('\\ce{\\bond{x}}');
+      assertLatex('\\ce{}');
```

Comment:
if the bond type is invalid shouldn't it be a parse failure? Don't we want `assertLatex('')`?